backend/packages/Upgrade/src/api/services/MoocletDataService.ts

```diff
@@ -199,21 +199,19 @@ export class MoocletDataService {
     return response;
   }
 
-  /* not yet implemented */
-
-  // public async getVersionForNewLearner(moocletId: number, userId: string) {
-  //   const endpoint = `/mooclet/${moocletId}/run?learner=${userId}`;
+  public async getVersionForNewLearner(moocletId: number, userId: string) {
```

Comment:
Is the return type (`Promise<MoocletVersionResponseDetails>`) inferred here?

Reply:
i can add a tyep.